cherryontech/jupiter-jumpers
# Daily

## Description

1. Creates a daily view for each log
2. Renders the daily view whenever the user clicks on a calendar date (in either the small or large calendar), or when they click "Today" in the main view.

## Related Issue
N/A

## Related Story Card

[SCRUM 28](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-28)
[SCRUM 27](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-27)

## Type of Changes
Feature update, and styling update.

## Acceptance Criteria

Given I am on the "Daily View" of a particular log
 When …
Then each log is displayed with:
- [X]  Date
- [X]  Weather icon
- [X]  Reflection content (if added)

- [X]   Given I am on the app's home screen or another section of the app
- [X]   When I press a mood icon in the calendar
- [X]    Then I am taken to that entry’s published view in the right sidebar

## Update Screenshots

_Provide screenshots of the view before and after the changes._

### Before

### After
![Screenshot 2024-05-19 at 9 14 29 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/e46acca7-6f15-4759-bc68-81939557fc3f)

## Testing Instructions

1. Click a day in the big calendar. Ensure its summary is correctly rendered.
2. Cick a day in the mini calendar. Ensure its summary is correctly rendered.
3. Click 'Today' in the main view. Ensure today's summary is correctly rendered.
4. Attempt the above for a new log entry. 
5. Ensure the collapsible sidebar works.

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
